python/py-musicbrainzngs/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-musicbrainzngs
+version             0.7.1
+revision            0
+
+categories-append   multimedia
+license             BSD
```

Comment:
license is listed by upstream as [`BSD-2`](https://github.com/alastair/python-musicbrainzngs/blob/8ed0187d487b54e2c81ad48ba72cbd4dd5ae9ace/setup.py#L22) - in MacPorts this corresponds to `BSD-old` (`BSD` refers to `BSD-3`).